Woof/Woof/NetworkLayer/Endpoint/WoofAppEndpoint+HTTPAPIEndpoint.swift

```diff
@@ -15,19 +15,22 @@ extension WoofAppEndpoint: HTTPAPIEndpoint {
 
     var path: String {
         switch self {
-        default: return ""
+        case .addNewSitter: return EndpointPath.addNewSitter
         }
     }
 
     var task: HTTPTask {
         switch self {
-        default: return .request
+        case let .addNewSitter(sitter):
+            // transform sitter to body parameters
+            let bodyParameters = try? sitter.asDictionary()
+            return .requestWithBodyParameters(bodyParameters: bodyParameters)
```

Comment:
Do we need to send a request if the `bodyParameters` is `nil`?

Reply:
It's just a configuration of an endpoint and describing a type of task, it's not a request method.